tests/tools/jira/test_jira_dryrun_client.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+from unittest.mock import Mock
+from tools.jira_client.jira_api import CLOSED_STATUS
+from tools.jira_client.jira_dryrun_client import JiraDryRunClient
+
+
+@pytest.fixture()
+def devnull_stream():
+    with open(os.devnull, "w") as devnull:
+        yield devnull
+
+
+def get_mock_issue(key="foobar"):
```

Comment:
apply same suggestion as in a different case (``Mock(key=key)``)
and let's just get rid of those ``get_mock_issue`` functions. they are redundant and can be replaced with the body of the function

Reply:
this applies here and elsewhere, unless the function is more complex like in ``get_mock_issue``